feat: update description of SINGLE_VIEW_PORTFOLIO_ORIGIN (#986)
mia-platform/documentation: docs/fast_data/configuration/single_view_creator/index.md

| SINGLE_VIEWS_DATABASE               | &check;  | The db from where single views are written.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 | -                   |
| SINGLE_VIEWS_COLLECTION             | &check;  | It must be equals to the Single View name the service is in charge of keeping updated.                                                                                                                                                                                                                                                                                                                                                                                                                                      | -                   |

> | SINGLE_VIEWS_PORTFOLIO_ORIGIN       | &check;  | name that will be used as an identifier for debugging and tracking of requests to be categorize metrics and be included in `sv-update` messages; it should represent the source of any request received by the Single View Creator (in case the SVC receives requests from one System of Record, we suggest to use the name of said SoR) 

Reviewer: ```suggestion
| SINGLE_VIEWS_PORTFOLIO_ORIGIN       | &check;  | Name that should represent the source of any request received by the Single View Creator (in case the SVC receives requests from one System of Record, we suggest to use the name of said SoR). This will be used as an identifier for debugging, tracking requests and categorizing metrics;
```
